File: src/OrganizationLookup.Server/Models/Organization.cs
using Newtonsoft.Json;

namespace OrganizationLookup.Server.Models
{
    public class Organization
    {
        public string OrgNumber { get; set; }
        public string BrregName { get; set; }
        public int NumberOfEmployees { get; set; }
        public string BusinessCategoryCode { get; set; }
        public string LegalStructureCode { get; set; }
        public bool IsBankrupt { get; set; }
        public bool IsDeleted { get; set; }
    }
}

File: src/organizationlookup.client/src/interfaces/organization.ts
interface Organization {
    orgNumber: string;
    name: string
    brregName: string | null;
    numberOfEmployees: number | null;
    businessCategoryCode: string | null;
    legalStructureCode: string | null;
    isBankrupt: boolean | null;
    isDeleted: boolean | null;
}

export type { Organization};